src/display/displayCommon.h

```diff
@@ -238,10 +238,10 @@ void displayLogo(String displaymessagetext, String displaymessagetext2) {
  * @brief display shot timer
  */
 void displayShottimer(void) {
-    if ((machineState == kBrew || brewSwitchTriggerCase == 31) && FEATURE_SHOTTIMER == 1 && SHOTTIMER_TYPE == 1) {
+    if ((machineState == kBrew || brewSwitchCase == 31) && FEATURE_SHOTTIMER == 1 && SHOTTIMER_TYPE == 1) {
```

Comment:
Not really the focus of this PR but can we also replace these magic numbers with a proper enum for the states like for the machineState?

Reply:
I thought about it for a few minutes and couldn't come up with good names. Suggestions welcome :D